docs-2.0/8.service-tuning/compaction.md

```diff
@@ -0,0 +1,63 @@
+# Compaction
+
+This document gives some information to compaction.
+
+## Introduction to compaction
+
+In Nebula Graph, compaction is the most important background process. Compaction has an important effect on performance.
+
+Compaction re-reads the data that is written on the hard disk, then re-organizes the data structure and the indexes to make the data more easier to read. The read performance can increase by times after compaction. Thus, to get high read performance, trigger compaction manually when writing a large amount of data into Nebula Graph. Note that compaction leads to long time hard disk IO, we suggest that you do compaction during business troughs (for example, early morning).
+
+Nebula Graph has these two type of compaction:
+
+- Automatic compaction
```

Comment:
I think it should be better if this is an h2 title. We might need to add the link of Automatic compaction and Manual compaction (Full compaction) in other topics, such as FAQ.